syncserver/src/main.rs

```diff
@@ -45,10 +45,12 @@ async fn main() -> Result<(), Box<dyn Error>> {
         // Note: set "debug: true," to diagnose sentry issues
         transport: Some(Arc::new(curl_transport_factory)),
         release: sentry::release_name!(),
+        environment: Some(Cow::Owned(settings.environment.clone())),
```

Comment:
I'll just note there's numerous ways to construct a `Cow`, including via various [From (and thus Into)](https://doc.rust-lang.org/rust-by-example/conversion/from_into.html) implementations for `Cow`, so another common way of doing this would be:
```suggestion
        environment: Some(settings.environment.clone().into()),
```

But the explicit way here is totally fine, I've often written it this way.

Reply:
Thanks for the protip.  Is there a preference for the repo/project?